Remove leading space from keywords.txt identifier

per1234 opened the pull request:
The Arduino IDE requires the use of a single true tab separator between the keyword name and identifier. When spaces are used rather than a true tab the keyword is not highlighted.

Reference:
https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5:-Library-specification#keywords

per1234 committed:
Remove leading space from keywords.txt identifier

Each field of keywords.txt is separated by a single true tab. Leading spaces on a keyword identifier causes it to not be recognized by the Arduino IDE. On Arduino IDE 1.6.5 and newer an unrecognized keyword identifier causes the default editor.function.style highlighting to be used (as with KEYWORD2, KEYWORD3).

Reference:
https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5:-Library-specification#keywords

sprilukin commented:
Hi, @per1234. Thanks for the pull request.
this change will cause the release of the new version, so some additional work should be done:
`CHANGELOG.md` and `library.json` files should be updated with next releasable version: `0.0.9`

If you want me to accept your pull request soon, please add these changes to your PR.

Another way is to create PR not to master but to another branch (`keywords-separator` is fine), so i can complete that release-related work before merging to master, but that will be slower.

per1234 committed:
Prepare 0.0.9 release

As requested at:
https://github.com/sprilukin/lib_BL999/pull/5#issuecomment-405597080

per1234 commented:
OK, I have now done as you instructed.

sprilukin commented:
@per1234 thank you. Pull request accepted and new version `0.0.9` has been released
